fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/LootJsonParser.java

```diff
@@ -26,28 +27,16 @@
 
 public final class LootJsonParser {
 	/* Reading this from LootManager to access all serializers from vanilla. */
-	private static final Lazy<Gson> GSON = new Lazy<>(() -> {
-		try {
-			Field gsonField = Stream.of(LootManager.class.getDeclaredFields())
-					.filter(field -> field.getType() == Gson.class)
-					.findFirst()
-					.orElseThrow(() -> new RuntimeException("Gson not found in LootManager!"));
-			gsonField.setAccessible(true);
-			return (Gson) gsonField.get(null);
-		} catch (Exception e) {
-			throw new RuntimeException("Exception while getting Gson instance from LootManager", e);
-		}
-	});
+	private static final Gson GSON = LootManagerAccess.getGson();
```

Comment:
Replacing a Lazy with a static init could be risky, if the LootManager's GSON object isn't initialized in the static initializer. Have you verified that this is valid?

Reply:
that is a static final field fyi.

Reply:
But it was a Lazy, meaning that the field wouldn't be accessed in the target class until one of the interface methods was actually invoked. I'm not saying that this is an actual issue, just a potential bug that needs to be checked.